src/interface/storage.thrift

```diff
@@ -203,6 +205,44 @@ struct CatchUpDataReq {
     3: common.HostAddr     target,
 }
 
+struct UpdateRespData {
+    1: bool upsert,
+    2: common.VertexID vertex_id,       // Only valid when update vertex
+    3: EdgeKey edge_key,                // Only valid when update edge
+    4: binary data,
+}
+
+struct UpdateResponse {
+    1: required ResponseCommon result,
+    2: optional common.Schema schema,   // return column related props
+    3: optional list<UpdateRespData> return_data,
```

Comment:
Why return a list of the response?

Do we support update batch of vertex in one sentence?

Reply:
yep, fixed!